src/common/i18n.js

```diff
@@ -0,0 +1,164 @@
+const defaultLanguage = 'en';
+
+const i18n = {
+	labels: {
+		en: {
+			advertisement: 'Advertisement',
+			'learn-more': 'Learn More'
```

Comment:
Should we request more translations for 'Learn more'? What about some follow up ticket

Reply:
Good idea, I will create a ticket.